source/base/ByteBuffer.ooc

```diff
@@ -87,62 +87,3 @@ _RecoverableByteBuffer: class extends ByteBuffer {
 		}
 	}
 }
-
-_RecyclableByteBuffer: class extends ByteBuffer {
```

Comment:
This reuse of memory was unreliable on other computers and covered up access violations and uninitialized memory from being detected by Valgrind.